src/dbwrapper.h

```diff
@@ -167,7 +166,7 @@ class CDBIterator
 
     template<typename V> bool GetValue(V& value) {
         try {
-            CDataStream ssValue{GetValueImpl(), SER_DISK, CLIENT_VERSION};
+            DataStream ssValue{GetValueImpl()};
```

Comment:
Just saw that this did replace the `CDataStream` with a `DataStream` here in `CDBIterator` and `CDBWrapper`, but it is still using a `CDataStream` for `CDBBatch`. Should that be replaced too, as I've done here: https://github.com/TheCharlatan/bitcoin/commit/95405b8a4f3816277c3dbf075c8da013226e6cec?

Reply:
Yes please, thanks for catching this. IIRC I couldn't do this with my original approach of converting to `SerParams`, but forgot about it after switching. As @MarcoFalke says, this must be correct if it compiles.